packages/subgraph/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@aragon/core-subgraph",
-  "version": "0.9.0-alpha",
+  "version": "0.10.0-alpha",
```

Are we publishing subgraph via NPM?

```suggestion
  "version": "0.10.0-beta",
```